rllib/models/tests/test_distributions.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import numpy as np
```

Comment:
Note that adding __init__ to the tests module messes with bazel resolution, you'll probably need to delete it.

Reply:
All tests/__init__.py files were removed.